ui/app/components/secret-engine/configure-aws.hbs

```diff
@@ -109,4 +121,29 @@
       />
     {{/if}}
   </div>
-</form>
\ No newline at end of file
+</form>
+
+{{#if this.saveIssuerWarning}}
+  <Hds::Modal @color="warning" @onClose={{action (mut this.saveIssuerWarning) ""}} data-test-issuer-warning as |M|>
+    <M.Header @icon="alert-circle">
+      Are you sure?
+    </M.Header>
+    <M.Body>
+      <p class="has-bottom-margin-s">
+        {{this.saveIssuerWarning}}
+      </p>
+    </M.Body>
+    <M.Footer>
+      <Hds::ButtonSet>
```

Comment:
No need to change, but just out of curiosity - is there a reason you didn't use the built-in footer here? For what it's worth, I think it allows you to call `F.close` which will call the `onClose` function above. I'm not sure of the difference stylistically
```hbs
<M.Footer as |F|>
   <Hds::Button type="button" @text="Confirm" {{on "click" F.close}} />
</M.Footer>
```

Reply:
I copied this from a different component, so that's the only reason